DateTimeInput/README.md

```diff
@@ -0,0 +1,77 @@
+# DateTimeInput Interface
+
+**Status**: Accepted
+
+| Name         | ID                                                                |
+| :--------    | :---------------------------------------------------------------- |
+| DateTimeInput| 4e862a9df81183ab425bdf0fbd76bd0b558c7f44c24887b4354bf1c26c74a623  |
+
+
+## Description
+
+Allows to input date and time.
+
+`getDate` - returns date timestamp selected by user.
+
+Note: function accepts date in unixtime (seconds from the midnight UTC-0 of 1st Jan 1970) which is int128 bits long, negative values are allowed. Date unixtime includes day, month, year and 0 hours, 0 minutes and 0 seconds.
+
+arguments:
```

Comment:
We've forgotten to pass `currentDate` as per the
https://www.notion.so/tonlabs/DateInput-TimeInput-proposals-890d07d2df8849f1a038f0d4be6dd937#db142b96a89040168cd38c1172b19e1d